sal_interfaces/FiberSpectrograph/FiberSpectrograph_Commands.xml

```diff
@@ -1,39 +1,51 @@
-    <Author></Author>
-    <EFDB_Topic>FiberSpectrograph_command_captureSpectImage</EFDB_Topic>
-    <Alias>captureSpectImage</Alias>
-    <Device></Device>
-    <Property></Property>
-    <Action></Action>
-    <Value></Value>
-    <Description>Capture a spectrum for integration time specified</Description>
-    <item>
-        <EFDB_Name>integrationTime</EFDB_Name>
-        <Description>The detector in the spectrometer can capture light for a length of time specified by integration time</Description>
-        <IDL_Type>float</IDL_Type>
-        <Units>second</Units>
-        <Count>1</Count>
-    </item>
-    <item>
-        <EFDB_Name>imageType</EFDB_Name>
-        <Description>Type of image. For example, Standard exposure.</Description>
-        <IDL_Type>string</IDL_Type>
-        <IDL_Size>256</IDL_Size>
-        <Units>unitless</Units>
-        <Count>1</Count>
-    </item>
-    <item>
-        <EFDB_Name>lamp</EFDB_Name>
-        <Description>Light used for measurement.</Description>
-        <IDL_Type>string</IDL_Type>
-        <IDL_Size>256</IDL_Size>
-        <Units>unitless</Units>
-        <Count>1</Count>
-    </item>
-</SALCommand></SALCommandSet>
+<SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
+    <SALCommand>
+        <Subsystem>FiberSpectrograph</Subsystem>
+        <Version>3.7.1</Version>
+        <Author/>
+        <EFDB_Topic>FiberSpectrograph_command_expose</EFDB_Topic>
+        <Alias>expose</Alias>
+        <Device/>
+        <Property/>
+        <Action/>
+        <Value/>
+        <Description>Capture an image from the spectrograph.</Description>
+        <item>
+            <EFDB_Name>duration</EFDB_Name>
+            <Description>The duration of the exposure to take.</Description>
+            <IDL_Type>float</IDL_Type>
+            <Units>second</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>type</EFDB_Name>
+            <Description>Type of image. What does this mean?????</Description>
+            <IDL_Type>string</IDL_Type>
+            <IDL_Size>256</IDL_Size>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+        <item>
+            <EFDB_Name>lamp</EFDB_Name>
+            <Description>Light used for measurement. Is this relevant???</Description>
+            <IDL_Type>string</IDL_Type>
+            <IDL_Size>256</IDL_Size>
+            <Units>unitless</Units>
+            <Count>1</Count>
+        </item>
+    </SALCommand>
+    <SALCommand>
+        <Subsystem>FiberSpectrograph</Subsystem>
+        <Version>3.7.1</Version>
+        <Author/>
+        <EFDB_Topic>FiberSpectrograph_command_cancelExposure</EFDB_Topic>
+        <Alias>cancelExposure</Alias>
+        <Device/>
+        <Property/>
+        <Action/>
+        <Value/>
+        <Description>Cancel an active exposure from the spectrograph.</Description>
```

Comment:
What happens if there is no active exposure? Does the command succeed or fail? It'd be nice to say here. (I suggest "succeed" so as to avoid race conditions).

Reply:
I've written the spectrograph python library such that `cancel` when there is no exposure is a no-op. I'll note that here.